src/test/resources/texts/texts_fr_FR.properties

```diff
@@ -1,2 +1 @@
-greeting=Bonjour!
-farewell=Au revoir!
\ No newline at end of file
+textoConfirmacion=�Espera! �Eres un ser humano?
```

Comment:
texts in the correct language

Reply:
Esos textos estan puestos asi a proposito para que falle el test y se muestren los asserts, capturas, etc